### PR TITLE
Replace existing code for controller authentication with the Ruby SDK

### PR DESCRIPTION
**Pull request by cilim**
PR for issue https://github.com/stormpath/stormpath-rails/issues/62

Currently there's one test failing in the controller authentication because of raising not specific errors in the Ruby SDK. Merge this PR after the Ruby SDK's error handling is properly refactored.

### CODE REVIEW

**Comment by cilim** [comment]
Pull request ready for reviewal @DamirSvrtan :)

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ gemspec
 
 gem 'sqlite3', '~> 1.3'
 gem 'sass-rails'
+gem 'stormpath-sdk'
```
Comment (DamirSvrtan) [inline Gemfile]: Please remove this as it should not be necessary.

File: app/services/stormpath/rails/controller_authentication/from_bearer_auth.rb [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-        ].freeze
-
-        def initialize(authorization_header)
-          @authorization_header = authorization_header
-        end
-
-        def authenticate!
-          begin
-            AccountFromAccessToken.new(bearer_access_token).account
-          rescue *RESCUE_CLASSES
```
Comment (DamirSvrtan) [inline app/services/stormpath/rails/controller_authentication/from_bearer_auth.rb]: Create a request spec called 'authentication_spec' where you ping the me route and you check to see if the errors raised from ControllerAuthentication get saved in the Controller. Failing specs first.

Comment (cilim) [inline app/services/stormpath/rails/controller_authentication/from_bearer_auth.rb]: Spec created @DamirSvrtan

**Pull request merged**